src/screens/Home/Home.js

```diff
@@ -56,6 +56,7 @@ import ChartsSection from './ChartsSection';
 import AssetsSection from './AssetsSection';
 import FloatingActions from './FloatingActions';
 import { useAccountCollectibleCounts } from './utils';
+import GovernanceCallBanner from '../GovernanceCall/GovernanceCallBanner';
```

Comment:
Can that be moved up as `components/GovernanceCall/GovernanceCallBanner`?

Reply:
Done